sprint/SPR_202306018/contenuti/previsione.tex

```diff
@@ -0,0 +1,7 @@
+\section{Preventivo}
+
+In questo sprint abbiamo preventivato di completare il secondo dei 4 incrementi definiti nel precedente sprint e di iniziare lo sviluppo del terzo.
+
+\subsection{Milestones:}  
+
+In questo sprint non sono state pianificate milestones.
```

Comment:
Ok, ma sta roba non va a nostro vantaggio, dovevamo pensare a più milestones, vabbè dai. lol

Reply:
Le ho create qualche settimana fa sui vari repository di GitHub, sui diari di bordo non sono state inserite né da me né da Luca. Se qualcuno dà un occhio nei vari repository che ci siano le milestones ed eventualmente le crea le posso mettere nei verbali

Reply:
Indicativamente pensavo di inserire le milestones nello sprint 18 dato che le ho create in quello sprint, altrimenti possiamo fare risalire a qualche sprint precedente e scrivere che in quelli successivi cercavamo di raggiungerle